src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx

```diff
@@ -1007,6 +1007,52 @@ This section defines the Node.js agent variables in the order they typically app
 
 This section allows you to choose which API methods are enabled. Each configuration option allows you to modularly enable API methods that are responsible for sending custom information to New Relic.
 
+<CollapserGroup>
+  <Collapser
+    id="custom-attributes"
+    title="esm.custom_instrumentation_entrypoint"
+  >
+    <table>
+      <tbody>
+        <tr>
+          <th>
+            Type
+          </th>
+
+          <td>
+            String
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            Default
+          </th>
+
+          <td>
+            (none)
+          </td>
+        </tr>
+
+        <tr>
+          <th>
+            [Environ variable](#environment)
+          </th>
+
+          <td>
+            `NEW_RELIC_API_ESM_CUSTOM_INSTRUMENTATION_ENTRYPOINT`
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+    Complete filepath to custom instrumentation definition, enables the ability to load custom instrumentation in ES module applications.
```

Comment:
I think it's the relative path from the application root

Reply:
We're using `path.resolve` which supports full and relative filepaths, right? Do you think I should change this to be something like: ```Filepath to custom instrumentation definition, which can be absolute or relative to the application root. This enables the ability...```

Reply:
realistically they won't be able to provide a full path due to the nature of apps starting. I would just say relative to where the application is started

Reply:
Wouldn't they be able to provide a full file path if they're using an env var? I'll change it to say relative, but flagging it as a potential source of confusion, which we can cross that bridge when we get to it.